.prettierrc

```diff
@@ -1,4 +1,4 @@
 {
   "singleQuote": true,
-  "trailingCommas": "es5"
+  "trailingComma": "es5"
```

Comment:
I would have expected other files to change. 

Can you run prettier on the whole project to fix everything?

Reply:
since prettier is only applied to the files that changed it didn't affect anything else, but makes sense to run it for the whole project

Reply:
The problem with changing the setting and not applying to all files is that the next 10+ PRs will have tons of line changes that are irrelevant to the PR. Instead, it is best to make all the formatting changes in one PR and avoid noise in the future.

Reply:
I'd rather apply it "iteratively" to not spam the git history. But I don't feel strongly if there's a better reason to apply it.

Reply:
(And also avoid releasing everything in the next canary)

Reply:
My argument is that applying iteratively is more spam, not less. It would make git blame have a commit message that is totally irrelevant for that line. Doing it all at once means one commit for all the lines that changed. Then never again. To me, that is less spam.

Reply:
Also consider non-ZEIT contributors who have no idea why their PR changed every line in a file when they only changed one line.

Reply:
Ok, that makes sense. And since we moved everything in this repo, it's probably the best time to do it.

Reply:
@AndyBitz See this PR for an example: https://github.com/zeit/now-builders/pull/563#issuecomment-498868054

Reply:
I've tried, but I think it is better to only change files whenever they got changed, otherwise it will be too many changes

Reply:
I've forgot it had `automerge`